### PR TITLE
create dummy data

### PR DESCRIPTION
**Pull request by 0x932b**
As a developer
I need data
So that I have a response to return to a request

Acceptance Criteria:
Code is committed to git
{
productA: [product1, product2, product3],
productB: [product4, product5, product6]
}

### CODE REVIEW

**Comment by 0x932b** [comment]
With the help of some online tools (www.json-generator.com, online-generator.com)

The scripts would generate a dummy json file with product name as it's object name,
also with 3 attributes: id, name, recommendations.
Recommendations is randomly assigned priority with a size range from 1 to 5

**Comment by jiweix** [comment]
I think we might want to include the product id in the recommendation list? If we use database for products, the product id is likely to be used to retrieve product information.

**Comment by sadam0930** [comment]
I agree! I was just mocking up what came to mind yesterday but totally
makes sense. Maybe the data looks like this?
GET /recommendation/123
Response =
{ id: 123,
Name: productA
Recommendations: [
{Id: 124,
Name: productB},
{Id: 125
Name: productC}
]}

So the dummy data is a list of maps like above?

**Comment by 0x932b** [comment]
+1 yeah I'll work on that.

@sadam0930 I think the current format in the pull request fit the data you are looking for, besides the naming conventions?
so a GET /recommendation/123 would be fetching with the key "123" for whatever in the {} in:
"123":{
 id:
 name:
 reccoms:[]
}
You guys want me to drop the priority part of the recommendations?

**Comment by sadam0930** [comment]
Yeah, I was just spewing thoughts again, but what you already have is
solid. I like the priority. Nothing wrong with more data.

File: dummy/scripts/getDummyRecomm.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    jsonFile = open(os.path.join(__location__, "json-generator.json"), "r")
+    products = json.load(jsonFile)
+    jsonFile.close()
+    return products
+
+
+def getProductList():
+    product_list_file = open(os.path.join(__location__, "dummy_product_names"), "r")
+    prod_list = [line.strip() for line in product_list_file]
+    product_list_file.close
```
Comment (0x932b) [inline dummy/scripts/getDummyRecomm.py]: should be product_list_file.close()

**Review by jiweix — APPROVED** [review]

**Pull request merged**